Security-Operations/L2-Security-Engineer.md

```diff
@@ -18,14 +18,14 @@
 <summary>Examples</summary>
 
 - I was nominated as the incident controller and successfully followed the relevant incident response plan.
-- When faced with a small project, I understood the brief the first time and was able to reach out myself to find the answers I needed to complete it.
-- I added a new module to the Beholder.
+- When faced with a small project, I understood the brief the first time and was comfortable reaching out to my peers to find the answers I needed to complete it.
```

Comment:
Hmmm. Understanding the brief is important... and getting the task completed is important... and getting help is important. Is this wording more clear?

```suggestion
- When faced with a small project, I understood the brief the first time and I completed the project, proactively reaching out to peers for help when I needed help or a sounding board.
```